collection-scripts/gather_namespaced_resources

```diff
@@ -20,7 +20,7 @@ resources=()
 # resources+=(storageclusters)
 # resources+=(storagesystem)
 
-# collect OB/OBC resoureces
+# collect OB/OBC resources
```

Comment:
There is a PR open for fixing this CI failure at: #164 

It would make sense to exclude this change from this PR (as it is unrelated anyways)

Reply:
okay sure, since ci was failing, I added it.